README.md

```diff
@@ -66,7 +66,8 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-The link will be added soon.
+- [Live Demo](pocketpal-vqre.onrender.com)
```

Comment:
- [ ] Please fix your live demo link , it is throwing a 404 error 


https://user-images.githubusercontent.com/75126481/236606395-b948ed77-ccc3-488c-b937-9e739e844532.mov